--- noyau.v1	2024-09-25 11:09:06.319030788 +0200
+++ noyau.v2	2024-09-25 11:09:06.319030788 +0200
@@ -22,8 +22,8 @@
 #
 
 VERSION = 2013
-PATCHLEVEL = 01
-SUBLEVEL = -rc1
+PATCHLEVEL = 02
+SUBLEVEL = -rc2
 EXTRAVERSION =
 ifneq "$(SUBLEVEL)" ""
 U_BOOT_VERSION = $(VERSION).$(PATCHLEVEL).$(SUBLEVEL)$(EXTRAVERSION)
@@ -51,5 +51,5 @@
 	else if [ -x /bin/bash ]; then echo /bin/bash; \
 	else echo sh; fi; fi)
 
-export	SHELL CC GCC
+export	SHELL 
 
